### ISSUE TITLE
Failed to run binner in Classify_bins module

### ISSUE THREAD

**Issue by Duannai** [bxlab/metaWRAP#205]
[binning.log](https://github.com/bxlab/metaWRAP/files/3373155/binning.log)
[high_classification.log](https://github.com/bxlab/metaWRAP/files/3373156/high_classification.log)
Please check the follow files, one is software log and the other is binning log.

It was normal before, when the minimum completion and contaminate were set as 90% and 10%, respectively. Now, they are set as 50% and 10%, respectively. 

So is it caused by some low completion?

**Comment by ursky** [comment]
No, there is probably a sequence that was only binned with a lowed completion minimum that produces an error in the classifier. Can you provide `predictions.gff3` so I can maybe see whats wrong with it?

**Comment by Duannai** [comment]
[predictions.gff3.zip](https://github.com/bxlab/metaWRAP/files/3373341/predictions.gff3.zip)
Please check the attach file.
